src/nft/components/cart/CartAssetRow.css.ts

```diff
@@ -0,0 +1,152 @@
+import { style } from '@vanilla-extract/css'
+import { sprinkles } from 'nft/css/sprinkles.css'
+
+export const cartAssetImage = style([
+  sprinkles({
+    borderRadius: '8',
+    height: '52',
+    width: '52',
+    marginRight: '12',
+    marginLeft: '8',
+    cursor: 'pointer',
+  }),
+  {
+    boxSizing: 'border-box',
+  },
+])
+
+export const cartAssetName = style([
+  sprinkles({
+    fontWeight: 'medium',
+    overflow: 'hidden',
+    marginRight: 'auto',
+  }),
+  {
+    marginTop: '4px',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+  },
+])
+
+export const cartAssetCollectionName = style([
+  sprinkles({
+    fontWeight: 'normal',
+    overflow: 'hidden',
+  }),
+  {
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    maxWidth: '65%',
+  },
+])
+
+export const cartAssetRowBottom = style([
+  sprinkles({
+    width: 'full',
+    display: 'flex',
+    flexWrap: 'nowrap',
+    marginRight: '4',
+  }),
+  {
+    marginTop: '-10px',
+  },
+])
+
+export const removeAsset = style({
+  position: 'absolute',
+  bottom: '-12px',
+  left: '22px',
+  cursor: 'pointer',
+  visibility: 'hidden',
+})
+
+export const assetHovered = style({
+  visibility: 'visible',
+})
+
+export const verifiedBadge = style({
+  height: '12px',
+  width: '12px',
+  marginLeft: '2px',
+  boxSizing: 'border-box',
+})
+
+export const removeIcon = style({
+  width: '32px',
+})
+
+export const ethIcon = style({
```

Comment:
don't think we need any of the styles below this since the old bag is getting deprecated